services/multi-cluster/ilb/README.md

```diff
@@ -0,0 +1,89 @@
+
+### Relevant documentation
+
+- [Shared VPC Concepts](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services)
+- [Setting Up Multi-cluster Services](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services)
+- [OSS Multi-cluster Services API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+
+#### Versions
+
+- GKE clusters on GCP
+- 1.17 and later versions of GKE supported
+- Tested and validated with v1.23.8-gke.1900 on Aug 23rd 2022
+
+### Networking Manifests
+
+This recipe demonstrates deploying a cluster (`gke-1`) and make it accessible to other cluster (`gke-2`). The Services in gke-1 are exposed via an Internal Load Balancer via a private IP address in the network. The pods in gke-2 will be able to communicate with gke-1 services via the internal ip address as they belong to the same network.
+
+![basic multi-cluster services](../../../images/internal-lb-service.png)
+
+
+
+### Try it out
+
+![Communication across clusters via ILB](../../../images/ilb-service-communication.png)
+
+
+1. Download this repo and navigate to this folder
+
+    ```sh
+    $ git clone https://github.com/GoogleCloudPlatform/gke-networking-recipes.git
+    Cloning into 'gke-networking-recipes'...
+
+    $ cd gke-networking-recipes/services/multi-cluster/ilb
+    ```
+
+2. Deploy the two clusters `gke-1` and `gke-2` as specified in [cluster setup](../../../cluster-setup.md)
+
+3. Now follow the steps for cluster registration with Hub and enablement of Multi-cluster Services.
+
+    There are two manifests in this folder:
+
+    - app.yaml is the manifest for the `whereami` Deployment and Service.
+    - ilb.yaml is the manifest for delpoying the whereami service with an Internal load balancer. NOTE: The ip address specified in ilb.yaml should be in the subnet CIDR range of the GKE cluster.
+
+4. Now log into `gke-1` and deploy the app.yaml manifest. You can configure these contexts as shown [here](../../../cluster-setup.md).
+
+    ```bash
+    $ kubectl --context=gke-1 apply -f app.yaml
+    namespace/multi-cluster-demo unchanged
+    deployment.apps/whereami created
+    service/whereami created
+
+    # Shows that pod is running and happy
+    $ kubectl --context=gke-1 get deploy -n multi-cluster-demo
+    NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+    whereami          1/1     1            1           44m
+    ```
+
+
+5. Now create the service as an ilb with an internal ip address within the CIDR of the subnet of GKE cluster.
+
+    ```bash
+    $ kubectl --context=gke-1 apply -f ilb.yaml
+    service/whereami-ilb created
+    ```
+
+
```

Comment:
Please, merge these two files in order to have only one single recipe.

Reply:
done and validated that it works